### PR TITLE
Add assessment groups schema

### PR DESCRIPTION
**Pull request by SWhitlam**

### CODE REVIEW

**Review by Coupey — APPROVED** [review]

File: src/main/resources/db/migration/V1_5__pre_sentence_assessment.sql [hunk 1]
```diff
@@ -1,3 +1,7 @@
+INSERT INTO assessment_schema (assessment_schema_id, assessment_schema_uuid, assessment_schema_code, oasys_assessment_type, oasys_create_assessment_at, assessment_name)
+VALUES
+(0, '51c2e87e-a540-4027-8f5a-e6c80511332f', 'ROSH', 'SHORT_FORM_PSR', 'START', 'Pre-Sentence Assessment');
+
```
Comment (adelaidaa) [inline src/main/resources/db/migration/V1_5__pre_sentence_assessment.sql:4]: mostly just this change

File: src/main/resources/db/migration/V1_7__rsr_only_assessment.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO assessment_schema (assessment_schema_id, assessment_schema_uuid, assessment_schema_code, oasys_assessment_type, oasys_create_assessment_at, assessment_name)
+VALUES
+(1, 'c3a6beac-37c0-46b6-b4b3-62086b624675', 'RSR_ONLY', 'SOMETHING_IN_OASYS', 'END', 'RSR only Assessment');
+
```
Comment (adelaidaa) [inline src/main/resources/db/migration/V1_7__rsr_only_assessment.sql:4]: and this

File: src/main/kotlin/uk/gov/justice/digital/assessments/services/QuestionService.kt [hunk 3]
```diff
@@ -158,22 +161,13 @@ class QuestionService(
   }
```
Comment (adelaidaa) [inline src/main/kotlin/uk/gov/justice/digital/assessments/services/QuestionService.kt:162]: @SWhitlam i have removed the coupling from then frontend endpoint to get groups by code and also uuid, this was necessary for the recursive load of groups so I have refactor that slightly so we just encapsulate getting the groups by uuid when needed in the service layer. I would like to merge this PR to test all of this is working as expected in the FE before i remove the top layer from the grouping table and we migrate the loading of the assessments to the new assessment schema layer.

**Review by adelaidaa — APPROVED** [review]
Approving after agreeing with @SWhitlam to merge and test this before we carry on with the replacement of groups by assessment schemas

**Pull request merged**